pkgs/development/libraries/libminc/default.nix

```diff
@@ -1,21 +1,30 @@
-{ stdenv, fetchurl, cmake, zlib, netcdf, hdf5 }:
+{ stdenv, fetchFromGitHub, cmake, zlib, netcdf, nifticlib, hdf5 }:
 
 stdenv.mkDerivation rec {
-  _name = "libminc";
-  name  = "${_name}-2.3.00";
+  pname = "libminc";
+  name  = "${pname}-2.3.01";
 
-  src = fetchurl {
-    url = "https://github.com/BIC-MNI/${_name}/archive/${_name}-2-3-00.tar.gz";
-    sha256 = "04ngqx4wkssxs9qqcgq2bvfs1cldcycmpcx587wy3b3m6lwf004c";
+  # current master significantly ahead of most recent release
+  src = fetchFromGitHub {
+    owner  = "BIC-MNI";
+    repo   = pname;
+    rev    = "e11c6df9321b4061bf87a7d43171ec55e9e3908f";
+    sha256 = "0lmd0js3jgni2mw1zfvd4qg6byxiv3ndgv2z3nm7975i83zw48xk";
   };
 
   nativeBuildInputs = [ cmake ];
-  buildInputs = [ zlib netcdf hdf5 ];
+  buildInputs = [ zlib netcdf nifticlib hdf5 ];
 
-  cmakeFlags = [ "-DBUILD_TESTING=${if doCheck then "ON" else "OFF"}"
-                 "-DLIBMINC_MINC1_SUPPORT=ON" ];
+  cmakeFlags = [ "-DBUILD_TESTING=${if doCheck then "TRUE" else "FALSE"}"
+                 "-DLIBMINC_MINC1_SUPPORT=TRUE"
+                 "-DLIBMINC_BUILD_SHARED_LIBS=TRUE"
+                 "-DLIBMINC_USE_SYSTEM_NIFTI=TRUE" ];
 
-  checkPhase = "ctest";
+
+  checkPhase = ''
+    export LD_LIBRARY_PATH="$(pwd)"  # see #22060
+    ctest -E ezminc_rw_test --output-on-failure   # ezminc_rw_test can't find libminc_io.so.5.2.0
+  '';
   doCheck = true;
 
   meta = with stdenv.lib; {
```

Comment:
Could you also add a `license = licenses.free;` field for this package too while you're at it?